### DISCUSSION TITLE
I want to display pure html content on the page

### DISCUSSION THREAD

**Discussion by SnowGuest** [nuxt/nuxt#17353] [Questions]
I found that when using nuxt3, there will be a script tag on the window that mounts the objects I bind through ref and html, I want to hide those js variables;
![image](https://user-images.githubusercontent.com/59474029/176118691-62e0593e-4191-48bb-b109-fc9c8e768f27.png)
I hope that these variables can be hidden, and I can no longer see them. Is there any way?

**Comment by Flanker72** [comment]
This data is passed from the server during the SSR process to make a correct hydration on the client. You may try to disable SSR features at all - such data shouldn't be injected in that case.

**Reply by SnowGuest** [reply]
Thanks for your answer, I'll try disabling the SSR function to hide the internal content
